kubernetes/template/DeploymentTemplates/MainTemplate.json

```diff
@@ -261,38 +261,17 @@
         "description": "Container runtime to deploy on each cluster node."
       },
       "type": "string"
+    },
+    "disconnectedAKSeURL": {
+      "type": "string",
+      "defaultValue": "",
+      "metadata": {
+        "description": "Azure Stack blob url to download AKS engine from a blob store in disconnected environment "
+      }
     }
   },
   "variables": {
-    "apiVersionDeployments": "2015-11-01",
-    "resourceGroupName": "[resourceGroup().name]",
-    "dnsNameForPublicIP": "[toLower(concat('vmd-dns', parameters('masterProfileDnsPrefix')))]",
-    "location": "[resourceGroup().location]",
-    "imagePublisher": "Canonical",
-    "imageOffer": "UbuntuServer",
-    "imageSku": "16.04-LTS",
-    "imageVersion": "latest",
-    "vmSize": "Standard_D2_v2",
-    "OSDiskName": "osdisk",
-    "nicName": "[concat('vmd-vnic', uniqueString(resourceGroup().id))]",
-    "addressPrefix": "10.0.0.0/24",
-    "subnetName": "mySubnet",
-    "subnetPrefix": "10.0.0.0/24",
```

Comment:
Why are we dropping so many variables

Reply:
Because these variables are not used in this template. The parameters are just passed on to another template so seems unreasonable to keep them there